documentation/cloudBasedFlow.md

```diff
@@ -1,22 +1,26 @@
-* `kettle.requests.request.handler.settings`, located in the `Settings.js` file. This will be referred to as `settings` in the below
-* * `gpii.request.flowManager.userToken` component in `FlowManagerUtitilities.js` - referred to in the below as `flowManagerUtilities`.
+This page describes the flow when GPII is run in cloud based flowmanager mode. This mainly involves two files:
+`CloudBasedFlowManager.js` - This will be referred to as `CloudBasedFlowManager` in the below
+* `FlowManagerUtitilities.js` - referred to in the below as `flowManagerUtilities`.
 
-The process of retrieving application settings in the cloudBased flowmanager mode is as follows:
-
-1. GET request is sent to the `:token/settings/:device` URL where `:token` is the user token to log in and `:device` is a device reporter payload, like: `{"OS":{"id":"web"},"solutions":[{"id":"org.chrome.cloud4chrome"}]}`. It is handled by the `handle` (settings) function which ensures the device payload is valid and fires two events: `onUserToken` and `onDevice`
-1. `onUserToken` event has one listener:
-  *`getPreferences` (flowManagerUtilities): which fetches the preferences and fires the `onPreferences` event when the preferences are fetched.
-1. `onDevice` event has one listener:
-** `getSolutions` (flowManagerUtilities): which fetches the solutions registry and filters it based on the device reporter info. The `onSolutionsRegistry` event is fired with the result.
-1. the `onReadyToMatch` event is listening to the three events described above: `onDevice`, `onPreferences` and `onSolutionsRegistry`. When these three events have been fired, the `onReadyToMatchEvent` will be fired.
-1. `onReadyToMatch` has one listener:
- `runMatchMakerFramework` (flowManagerUtilities): which kicks off the matchMaker framework. For the purpose of describing the overall login flow, it suffices to say that when the matchmaker framework has finished the matchmaking process, the `onMatch` event is called on the flow manager. For more details on the match maker frameworks internal workings, see: [Match Maker Framework Documentation](MatchMakerFramework.md)
-1. `onMatch` is being listened to by `runContextManager` (settings) which in turns fires up the (Context Manager)[ContextManager.md] which evaluates the Match Maker Frameworks output and fires the `onReadyForLifecycle` event.
-1. `onReadyForLifecycle` is being listened to by `processMatch` (settings) which applies the settings to the system.
+### Overview and APIs
 
+The cloudbased flowmanager can be considered as a standalone, cloud-only configuration of the GPII. This is relevant for applications that are built to support this, and running on a system without the core GPII installed. So for example, a simple javaphone that has a built-in support for GPII, but is not running the core architecture. This can send a user token and platform information to the cloudbased flowmanager, and get instructions back on how it should be set up.
```

Comment:
Check consistency throughout this file - sometimes we have "cloud based flowmanager" and sometimes "cloudBased flowManager" and then a few other variants